sw/airborne/firmwares/rotorcraft/guidance/guidance_hybrid.c

```diff
@@ -202,8 +202,8 @@ void guidance_hybrid_airspeed_to_attitude(struct Int32Eulers *ypr_sp)
 
     // 2) calculate roll/pitch commands
     struct Int32Vect2 hover_sp;
-    if (norm_sp_airspeed > (4 <<
-                            8)) { //if the setpoint is beyond 4m/s but the ref is not, the norm of the hover sp will stay at 4m/s
+    //if the setpoint is beyond 4m/s but the ref is not, the norm of the hover sp will stay at 4m/s
+    if (norm_sp_airspeed > (4 << 8)) {
```

Comment:
If this bit shift is a POS_FLOAT_OF_BFP, the macro should be used, same for lines below.